src/main/java/chanceCubes/network/CCubesClientPayloadHandler.java

```diff
@@ -0,0 +1,39 @@
+package chanceCubes.network;
+
+import chanceCubes.tileentities.TileChanceD20;
+import net.minecraft.client.Minecraft;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.neoforged.neoforge.network.handling.IPayloadContext;
+
+public class CCubesClientPayloadHandler
+{
+	private static final CCubesClientPayloadHandler INSTANCE = new CCubesClientPayloadHandler();
+
+	public static CCubesClientPayloadHandler getInstance()
+	{
+		return INSTANCE;
+	}
+
+	public void handleTrigger(final PacketTriggerD20 msg, final IPayloadContext context)
+	{
+		context.enqueueWork(() -> {
+					//Open Captcha Screen
```

Comment:
Take it this was from whatever you copied? xD

Reply:
Whoops